mslib/mscolab/server.py

```diff
@@ -500,6 +500,31 @@ def get_operation_details():
     return json.dumps(fm.get_operation_details(int(op_id), user))
 
 
+@APP.route('/set_counter', methods=["POST"])
```

Comment:
from nameing of this I think /set_state is better

counter is mostly understood as enumated number